pkg/controller/migmigration/migmigration_controller.go

```diff
@@ -120,14 +121,18 @@ func (r *ReconcileMigMigration) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	// Do the migration.
-	err = r.migrate(migration)
+	requeue, err := r.migrate(migration)
```

Comment:
Maybe at some point we'll want to pass a delay in seconds, but I think defaulting to 5 meets our needs well for now.